templates/corporate/upgrade.html

```diff
@@ -3,7 +3,7 @@
 
 {% block customhead %}
 <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-<script src="https://checkout.stripe.com/checkout.js"></script>
+<script src="https://js.stripe.com/v3/"></script>
 {% endblock %}
```

Comment:
We’ve concluded that we probably don’t need Stripe.js; leaving a reminder to remove this `customhead` block (also on `billing.html`, `event_status.html`).

Reply:
Thanks for the reminder. I have removed them.